deploy/olm-catalog/konveyor-operator/v1.2.2/konveyor-operator.v1.2.2.clusterserviceversion.yaml

```diff
@@ -31,7 +31,8 @@ metadata:
               "restic_timeout": "1h",
               "mig_pv_limit": "100",
               "mig_pod_limit": "100",
-              "mig_namespace_limit": "10"
+              "mig_namespace_limit": "10",
+              "mig_failure_rollback: false"
```

Comment:
@jmontleon can you ack these CSV changes? Want to make sure I hit all of them.

Reply:
ACK.